README.md

```diff
@@ -1,148 +1,168 @@
-# A Kubernates Cloud-Shell (Web Terminal) Operator.
+# A Kubernates Cloud Shell (Web Terminal) Operator
+
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6301/badge)](https://bestpractices.coreinfrastructure.org/projects/6301)
 
 English | [简体中文](https://github.com/cloudtty/cloudtty/blob/main/README_zh.md)
 
-# Why needs cloudtty ?
+cloudtty is an easy-to-use operator to run web terminal and cloud shell intended for a kubernetes-native environment.
+You can easily open a terminal on your own browser via cloudtty.
+The community is always open for any contributors and those who want to have a try.
+
+Early user terminals connected to computers were electromechanical teleprinters or teletypewriters (TeleTYpewriter, TTY), which might be the origin of TTY.
```

Comment:
Is it too many words to explain `TTY` ?
or you can put a separated chapter with title `Why names it as TTY? `

Reply:
Give a note to `what is tty`, which is searched out by google when I don't know. I'm a user of your cloudtty.